static/js/click_handlers.js

```diff
@@ -171,11 +171,43 @@ $(function () {
 
     $("#main_div").on("click", ".message_reaction", function (e) {
         e.stopPropagation();
+        $(this).popover('hide');
         var local_id = $(this).attr('data-reaction-id');
         var message_id = rows.get_message_id(this);
         reactions.process_reaction_click(message_id, local_id);
     });
 
+    $('#main_div').on('mouseenter', '.message_reaction', function (e) {
+        e.stopPropagation();
+        var elem = $(this);
+        var local_id = elem.attr('data-reaction-id');
+        var message_id = rows.get_message_id(this);
+        var title = reactions.generate_popover(message_id, local_id);
```

Comment:
I think the name `generate_popover()` is misleading since it is actually creating only the popover content, not the popover itself, right?